assets/src/edit-story/components/canvas/framesLayer.js

```diff
@@ -48,7 +48,7 @@ const DesignSpaceGuideline = styled.div`
   position: absolute;
   pointer-events: none;
   z-index: 1;
-  visibility: hidden;
+  visibility: ${({ isVisible }) => (isVisible ? 'visible' : 'hidden')};
```

Comment:
Hmm, can you check if S-C only uses two classes for this or generates a new class every time it toggles? If it's only ever two classes, this is fine. Otherwise, we should use `.attr({style:{...}})` to avoid generating infinite classes.

Reply:
Good point, didn't think about that! Fortunately, it's smart and uses two classes only.